src/Sylius/Bundle/ApiBundle/Serializer/ProductNormalizer.php

```diff
@@ -42,8 +43,12 @@ public function normalize($object, $format = null, array $context = [])
         $context[self::ALREADY_CALLED] = true;
 
         $data = $this->normalizer->normalize($object, $format, $context);
-        $variant = $this->defaultProductVariantResolver->getVariant($object);
-        $data['defaultVariant'] = $variant === null ? null : $this->iriConverter->getIriFromItem($variant);
+
+        $variantsIris = array_map(fn(ProductVariantInterface $variant): string => $this->iriConverter->getIriFromItem($variant), $object->getEnabledVariants()->toArray());
+        $data['variants'] = array_values($variantsIris);
```

Comment:
this should be equivalent:
```suggestion
        $data['variants'] = $object
            ->getEnabledVariants()
            ->map(fn(ProductVariantInterface $variant): string => $this->iriConverter->getIriFromItem($variant))
            ->getValues()
        ;
```

and seems to me to be more object-oriented and more readable